e2e/tests/api/forms/forms.e2e.spec.js

```diff
@@ -0,0 +1,69 @@
+ * "License"); you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * http://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ *
+ * Open MCT includes source code licensed under additional open source
+ * licenses. See the Open Source Licenses file (LICENSES.md) included with
+ * this source code distribution or the Licensing information page available
+ * at runtime from the About dialog for additional information.
+ *****************************************************************************/
+
+/*
+This test suite is dedicated to tests which verify the basic operations surrounding conditionSets.
+*/
+
+const { test, expect } = require('@playwright/test');
+
+test.describe('forms set', () => {
+    test('New folder form has title as required field', async ({ page }) => {
+        //Go to baseURL
+        await page.goto('/', { waitUntil: 'networkidle' });
+
+        // Click button:has-text("Create")
+        await page.click('button:has-text("Create")');
+        // Click :nth-match(:text("Folder"), 2)
+        await page.click(':nth-match(:text("Folder"), 2)');
+        // Click text=Properties Title Notes >> input[type="text"]
+        await page.click('text=Properties Title Notes >> input[type="text"]');
+        // Fill text=Properties Title Notes >> input[type="text"]
+        await page.fill('text=Properties Title Notes >> input[type="text"]', '');
+        // Press Tab
+        await page.press('text=Properties Title Notes >> input[type="text"]', 'Tab');
+        // Click text=OK Cancel
+        await page.click('text=OK', { force: true });
+
+        const okButton = page.locator('text=OK');
+
+        await expect(okButton).toBeDisabled();
+        await expect(page.locator('.c-form-row__state-indicator').first()).toHaveClass(/invalid/);
+
+        // Click text=Properties Title Notes >> input[type="text"]
+        await page.click('text=Properties Title Notes >> input[type="text"]');
+        // Fill text=Properties Title Notes >> input[type="text"]
+        await page.fill('text=Properties Title Notes >> input[type="text"]', 'test folder');
+        // Press Tab
+        await page.press('text=Properties Title Notes >> input[type="text"]', 'Tab');
+
+        await expect(page.locator('.c-form-row__state-indicator').first()).not.toHaveClass(/invalid/);
+
+        // Click text=OK
+        await Promise.all([
+            page.waitForNavigation(/*{ url: 'http://localhost:8080/#/browse/mine/617c17fc-30ea-4f94-96a9-ef7ce00e40af?tc.mode=fixed&tc.startBound=1643673151453&tc.endBound=1643674951453&tc.timeSystem=utc&view=grid' }*/),
```

Comment:
What's going on here, why do we have a commented out URL?

Reply:
also noticed in old test `condition.e2e.spec.js` removing